volontulo/templates/offers/offer_apply.html

```diff
@@ -14,7 +14,7 @@
             </div>
             <label class="col-xs-1 control-label" for="{{ form.email.id_for_label }}">Email:</label>
             <div class="col-xs-4">
-               <input type="text" value="{{ request.POST.email|default:volunteer_user.user.email }}" name="email" class="form-control" id="{{form.email.id_for_label }}"{% if user.is_authenticated %}disabled="disabled"{% endif %} />
```

Comment:
naa, minifying code is something else. 
If you really need the value, which is same as its attribute, that's really no prob. 
Just saying, it's not really that important in html5 ;)